transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/wizards/JavaPage.java

```diff
@@ -151,14 +152,13 @@ public void widgetSelected(final SelectionEvent event) {
                             model.setTargetType(ModelType.CLASS);
                             model.setTargetFilePath(selected.getFullyQualifiedName());
                         }
-                        final IType inner = selected;
 
                         UIJob uiJob = new UIJob(Messages.JavaPage_jobName_openError) {
                             @Override
                             public IStatus runInUIThread(IProgressMonitor monitor) {
                                 NewTransformationWizard wizard = (NewTransformationWizard) getWizard();
                                 try {
-                                    Class<?> tempClass = wizard.getLoader().loadClass(inner.getFullyQualifiedName());
+                                    Class<?> tempClass = wizard.loader.loadClass(selected.getFullyQualifiedName());
```

Comment:
Because previously, when the project was unknown or modifiable, there used to be a getter that had more than a single line of logic.  But now I do notice that there are several variables in that wizard marked as public that should be immutable after the wizard is initialized.  I'll fix that.